pkg/cmd/extension/manager_test.go

```diff
@@ -720,26 +752,31 @@ func TestManager_UpgradeExtension_GitExtension_Pinned(t *testing.T) {
 }
 
 func TestManager_Install_local(t *testing.T) {
-	extManagerDir := t.TempDir()
```

Comment:
Renamed this to be inline with all of the other tests but also because it is called `dataDir` within the extension manager, which now has 2 directories to care about.